README.md

```diff
@@ -74,6 +74,24 @@ https://ah-fulldeck-staging.herokuapp.com/api/v1/docs
 
    `export EMAIL_PORT=587`
 
+   `export GOOGLE_OAUTH2_KEY='Valid google key'`
+    
+   `export GOOGLE_OAUTH2_SECRET='Valid google app secret'`
+
+   `export FACEBOOK_KEY='Valid facebook key'`
```

Comment:
I don't know how to get these keys, could you maybe provide in your PR description for my testing purposes

Reply:
I have the keys but they are for my google, facebook and twitter accounts. I wanted you to create generate your own access tokens from the links in the PR but putting my keys here is a security issue.